src/_posts/aframe-v0.3.0.md

```diff
@@ -0,0 +1,130 @@
+---
+title: A-Frame v0.3.0 - Walk in a Web Page
+author: twitter|andgokevin|Kevin Ngo
+date: 2016-07-25 10:00:00
+layout: blog
+image:
+  src: aframe-v0.3.0.png
+---
+
+**v0.3.0** has dropped! *What is virtual reality without our legs nor our
+hands?* With support for the **[new WebVR 1.0 API][webvr1.0]**, we aim higher
+towards state of the art VR. With room-scale tracking, we can walk in a web
+page, and with tracked controller support, we can reach out into the world.
+
+<!-- more -->
+
+## What's New?
+
+**WebVR 1.0** support is the highlight of this release. This includes:
+
+- Better **room scale** support to be able to walk/dash/duck/jump in a web
+  page, notably with the HTC Vive.
+- **Link traversal** to be able to seamlessly travel from one VR experience to the next, just
+  as we would on normal webpages.
+- Hitting **90 frames per second** due to being able to push content directly to the headset
+  display rather than mirroring a desktop display. This also allows us to have different
+  content on the desktop display than the headset, opening the door for asynchronous gameplay
+  or spectator modes.
+
+As a corollary, A-Frame supports **tracked controllers**, notably with the HTC
+Vive. While part of the Gamepad API instead of the WebVR API, tracked
+controllers goes together with room scale; just like how our arms go
```

Comment:
> tracked controllers goes together with room scale

Change "goes" to "go". Also, this sentence is a little awkward IMO; is it necessary to mention Gamepad API vs WebVR API?